assets/css/style.css

```diff
@@ -15,22 +15,27 @@ ul {
     width: 100%;
     height: 100%;
 }
-.multi-columns .col-20 {
+.multi-columns__col-20 {
     float: left;
     width: 20%;
     height: 100%;
 }
-.multi-columns .col-80 {
-    float:left;
-    width: 80%;
+.multi-columns__col-30 {
+    float: left;
+    width: 30%;
     height: 100%;
 }
-.multi-columns .col-50 {
+.multi-columns__col-50 {
     float:left;
     width: 50%;
     height: 100%;
 }
-.multi-columns .pull-content-right {
+.multi-columns__col-80 {
+    float:left;
+    width: 80%;
+    height: 100%;
+}
+.multi-columns__pull-content-right {
```

Comment:
Please consider to use pull-content-right as stand-alone class. so we did not need multi-columns as prefix